src/test/java/com/wikia/webdriver/common/core/configuration/Configuration.java

```diff
@@ -285,4 +287,26 @@ public static Dimension getBrowserSize() {
   public static String getDisableCommunityPageSalesPitchDialog() {
     return getProp("disableCommunityPageSalesPitchDialog");
   }
+
+  public static HttpHost getBorderProxy() {
+
+    EnvType envType = getEnvType(getEnv());
+    File configFile = new File(getCredentialsFilePath());
+
+    switch (envType) {
+      case DEV: {
+        return new HttpHost(
+            XMLReader.getValue(configFile, "border.poz.address"),
+            Integer.parseInt(XMLReader.getValue(configFile, "border.poz.port")),
+            XMLReader.getValue(configFile, "border.poz.protocol")
+        );
+      }
+      default: {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Extract this nested code block into a method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1199)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Reduce this switch case number of lines from 6 to at most 5, for example by extracting code into methods. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1151)